magpie/adapter/__init__.py

```diff
@@ -138,6 +139,7 @@ def owsproxy_config(self, container):
 
     def configurator_factory(self, container):
         settings = get_settings(container)
+        set_cache_regions_from_settings(settings)
```

Comment:
Probably we should also add this call inside `magpie.app.main` since `@cache_region` affects generic services.